neuror/cut_plane/detection.py

```diff
@@ -205,48 +204,29 @@ def to_json(self):
             - details: A dict currently only containing -LogP of the bin where the cut plane was
                      found
         '''
-        return {'cut-leaves': self.cut_leaves_coordinates,
+        return {'cut-leaves': self.cut_leaves_coordinates if self.status == 'ok' else None,
                 'status': self.status,
-                'details': {'-LogP': self.minus_log_prob, 'bin-width': self.bin_width},
-                'cut-plane': super().to_json()}
-
-    def histogram(self):
-        '''Get the point distribution projected along the normal to the plane
-
-        Returns:
-            a numpy.histogram
-        '''
-        points = _get_points(self.morphology)
-        projected_points = self.project_on_directed_normal(points)
-        min_, max_ = np.min(projected_points, axis=0), np.max(projected_points, axis=0)
-        binning = np.arange(min_, max_ + self.bin_width, self.bin_width)
-        return np.histogram(projected_points, bins=binning)
+                'details': {'prob': self.prob, 'bin-width': self.bin_width},
```

Comment:
This looks like a breaking change so it should at least be added explicitly in the changelog.

Reply:
yes, this whole PR is very breaking change!